src/framework/preferences/js/PrefsEditor.js

```diff
@@ -391,13 +391,17 @@ var fluid_2_0 = fluid_2_0 || {};
             fluid.model.diff(userSelections.preferences, fluid.get(that.initialModel, ["preferences"]), stats);
 
             fluid.each(stats.changeMap, function (state, path) {
-                fluid.set(changedSelections, ["preferences", path], userSelections.preferences[path]);
+                fluid.set(changedPrefs, path, userSelections.preferences[path]);
```

Comment:
So i think this comment is at the wrong spot in the code. I guess i want an option where we just save changedPrefs and not the entire userSelections. Before I believe something like {contrast: "bw"} would have been saved, but in the current implementation "{preferences: {contrast: "bw"}}" would be saved. However, that will require existing implementations to change their data structure even though they aren't saving any other material.